{"uid":"ec67a56bafff84d9","name":"test_log_rotation","fullName":"tests.integration.test_log_rotation#test_log_rotation","historyId":"cee6faeba82de98eaf891ec9dd9520e1","time":{"start":1725158033848,"stop":1725158936345,"duration":902497},"description":"Test log rotation.","descriptionHtml":"<p>Test log rotation.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1788, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_log_rotation.py\", line 81, in test_log_rotation\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1040, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 131, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1725158033599,"stop":1725158033599,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1725158033599,"stop":1725158033847,"duration":248},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1725158033599,"stop":1725158033599,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1725158033847,"stop":1725158033847,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test log rotation.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1788, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_log_rotation.py\", line 81, in test_log_rotation\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1040, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 131, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","steps":[],"attachments":[{"uid":"f39b8fccea28f2b7","name":"log","source":"f39b8fccea28f2b7.txt","type":"text/plain","size":3714}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1725158936396,"stop":1725158936396,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1725158936397,"stop":1725158944047,"duration":7650},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1725158944048,"stop":1725158944048,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1725158944048,"stop":1725158944048,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1725158944048,"stop":1725158944048,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_log_rotation"},{"name":"host","value":"fv-az693-429"},{"name":"thread","value":"12371-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_log_rotation"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"dc53a4277e3c6241","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":29,"skipped":0,"passed":63,"unknown":3,"total":95},"items":[{"uid":"4bf97ce877b33d7a","reportUrl":"../238//#testresult/4bf97ce877b33d7a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1725070764412,"stop":1725071668762,"duration":904350}},{"uid":"af3966da6576adc9","reportUrl":"../236//#testresult/af3966da6576adc9","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724984505408,"stop":1724985407345,"duration":901937}},{"uid":"babcb7ba3e9364b3","reportUrl":"../233//#testresult/babcb7ba3e9364b3","status":"unknown","time":{}},{"uid":"bf62eb1ed842a6dd","reportUrl":"../231//#testresult/bf62eb1ed842a6dd","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724811587733,"stop":1724812490625,"duration":902892}},{"uid":"d9db84095b2111a4","reportUrl":"../229//#testresult/d9db84095b2111a4","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724725124835,"stop":1724726027728,"duration":902893}},{"uid":"451dbd2973ec25b0","reportUrl":"../227//#testresult/451dbd2973ec25b0","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724638676114,"stop":1724639578845,"duration":902731}},{"uid":"61706954e4a51ed0","reportUrl":"../226//#testresult/61706954e4a51ed0","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724552531030,"stop":1724553433381,"duration":902351}},{"uid":"cc2205f12a4d6673","reportUrl":"../225//#testresult/cc2205f12a4d6673","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724465751935,"stop":1724466654741,"duration":902806}},{"uid":"b18d2108d69164e3","reportUrl":"../224//#testresult/b18d2108d69164e3","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724379539983,"stop":1724380443531,"duration":903548}},{"uid":"e27c359bef5e4993","reportUrl":"../222//#testresult/e27c359bef5e4993","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724293127665,"stop":1724294030717,"duration":903052}},{"uid":"b8c41294eb855ebd","reportUrl":"../219//#testresult/b8c41294eb855ebd","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724206607332,"stop":1724207510094,"duration":902762}},{"uid":"47b995f7fe644461","reportUrl":"../216//#testresult/47b995f7fe644461","status":"unknown","time":{}},{"uid":"e1a0977e10764b46","reportUrl":"../214//#testresult/e1a0977e10764b46","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1724033938122,"stop":1724034841201,"duration":903079}},{"uid":"2bb12833884cbbf2","reportUrl":"../213//#testresult/2bb12833884cbbf2","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723947617742,"stop":1723948520612,"duration":902870}},{"uid":"89662081ac68e45b","reportUrl":"../212//#testresult/89662081ac68e45b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723860800389,"stop":1723861702695,"duration":902306}},{"uid":"159cb55469b1b61a","reportUrl":"../210//#testresult/159cb55469b1b61a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723774521112,"stop":1723775423616,"duration":902504}},{"uid":"7631f9aa8ba1a6a","reportUrl":"../209//#testresult/7631f9aa8ba1a6a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723687890504,"stop":1723688793601,"duration":903097}},{"uid":"93bcf1748a73610e","reportUrl":"../207//#testresult/93bcf1748a73610e","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723601995921,"stop":1723602898668,"duration":902747}},{"uid":"349db1f2fdc80c6b","reportUrl":"../205//#testresult/349db1f2fdc80c6b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723515688311,"stop":1723516590653,"duration":902342}},{"uid":"3a1d6d7b8a2be38a","reportUrl":"../204//#testresult/3a1d6d7b8a2be38a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1723429341116,"stop":1723430244519,"duration":903403}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"ec67a56bafff84d9.json","parameterValues":[]}